src/mds/MDBalancer.cc

```diff
@@ -36,8 +37,9 @@ using std::map;
 using std::vector;
 
 #include "common/config.h"
+#include "common/errno.h"
 
-#define dout_subsys ceph_subsys_mds
+#define dout_subsys ceph_subsys_mds_balancer
```

Comment:
I'm not sure we want this: iirc the --debug-mds flag isn't a prefixish thing, so this change will silence all the balancer and migrator logs for people who currently set --debug-mds